app/views/projects/identifier.html.erb

```diff
@@ -51,7 +51,7 @@ See docs/COPYRIGHT.rdoc for more details.
       </span>
       <span>
         <%= link_to settings_generic_project_path(@project), class: 'button' do %>
-          <span class="button--text"><%= l(:button_cancel) %></span>
+          <span class="button--text button -with-icon icon-cancel"><%= l(:button_cancel) %></span>
```

Comment:
This is a special case as the `button` class  is already set in line 53. The structure is here a bit different with the additional span and the class `button--text`. 
Please compare to [this place](https://github.com/opf/openproject/blob/dev/app/views/wiki/show.html.erb#L51) on how the structure should be.